testutil/validatormock/validatormock.go

```diff
@@ -31,12 +32,15 @@ import (
 	"github.com/obolnetwork/charon/tbls/tblsconv"
 )
 
-// Eth2AttProvider defines the eth2 beacon api providers required to perform attestations.
-type Eth2AttProvider interface {
+// Eth2Provider defines the eth2 beacon api providers required to perform attestations.
+type Eth2Provider interface {
```

Comment:
could maybe split this into two smaller interfaces, one for attestations and one for proposals